modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -9173,6 +9173,102 @@ type SpectroscopyConfigOptionFlamingos2 {
   filter: Flamingos2Filter!
 }
 
+"""
+Edit or create imaging science requirements
+"""
+input ImagingScienceRequirementsInput {
+
+  """
+  minimumFov, which may be unset by assigning a null value, or ignored by
+  skipping it altogether.
+  """
+  minimumFov: AngleInput
+
+  """
+  narrowFilters, which may be unset by assigning a null value, or ignored by
+  skipping it altogether.
+  """
+  narrowFilters: Boolean
+
+  """
+  broadFilters, which may be unset by assigning a null value, or ignored by
+  skipping it altogether.
+  """
+  broadFilters: Boolean
+
+  """
+  combinedFilters, which may be unset by assigning a null value, or ignored by
+  skipping it altogether.
+  """
+  combinedFilters: Boolean
+
+  """
+  gmosNorth, which may be unset by assigning a null value, or ignored by
+  skipping it altogether.
+  """
+  gmosNorth: ImagingScienceRequirementsGmosNorthInput
```

Comment:
This is debatable, whether we should include mode specific properties in the requirements.

Reply:
Having this here to allow the UI to retain row selections between visits/page refreshes will also mean that 2 people looking at the same observation will be changing the selections for each other. I'm not sure if that is good or bad. It seems a little weird, but then I guess editing the requirements changes the list of available observations and could have the side effect of changing the selections.

We will have to maintain this for each new imaging mode, though.

Reply:
I agree it is debatable. Well, the main use is to keep the selection between reloads, but maybe that's not that important?
We could remove it

Reply:
After some discussions we decided to remove those fields